presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/HiveCluster.java

```diff
@@ -26,6 +28,7 @@
 {
     /**
      * Create a connected {@link HiveMetastoreClient} to this HiveCluster
+     * @throws TException
```

Comment:
Per [Javadoc conventions](https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format), there should be a blank line before the tag list (in this case the `@throws` clause).
> Insert a blank comment line between the description and the list of tags, as shown.

We also don't add empty Javadoc declarations as they just clutter source code. We should add a description explaining when it would throw (or leave it out entirely as the reason is obvious given the role of the method):
```
@throws TException if there is an error connecting
```